src/rpcrawtransaction.cpp

```diff
@@ -343,6 +344,8 @@ UniValue createrawtransaction(const UniValue& params, bool fHelp)
             "      ...\n"
             "    }\n"
             "3. locktime                (numeric, optional, default=0) Raw locktime. Non-0 value also locktime-activates inputs\n"
+            "4. opt into RBF            (boolean, optional, default=false) Allow this transaction to be replaced by a transaction with heigher fees\n"
+
```

Comment:
It would also be less confusing what `rbfOptIn` does in light of it being disabled on the CLI too.